app/controllers/admin/products_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Admin::ProductsController < Admin::BaseController
+  before_action :set_admin_product, except: %i(index new create)
+
+  def index
+    @admin_products = Product.order_created_at
+    @pagy, @admin_products = pagy @admin_products
+  end
+
+  def show
+    @admin_product_details = @admin_product.product_details
+    @pagy, @admin_product_details = pagy @admin_product_details
+  end
+
+  def new
+    @admin_product = Product.new
+  end
+
+  def edit; end
+
+  def create
+    @admin_product = Product.new admin_product_params
```

Comment:
```suggestion
    @product = Product.new admin_product_params
```
ri thôi em, thêm admin làm j thế

Reply:
Để em sửa lại hết